hpat/tests/test_series.py

```diff
@@ -556,6 +556,37 @@ def test_series_op8(self):
             B = pd.Series(np.arange(n)**2)
             pd.testing.assert_series_equal(hpat_func(A, B), test_impl(A, B), check_names=False)
 
+    @unittest.skipIf(platform.system() == 'Windows', "Attribute dtype are different: int64, int32")
+    def test_series_op8_integer_scalar(self):
+        comparison_methods = ('lt', 'gt', 'le', 'ge', 'ne', 'eq')
+
+        for method in comparison_methods:
+            test_impl = _make_func_use_method_arg1(method)
+            hpat_func = hpat.jit(test_impl)
+
+            n = 11
+            operand_series = pd.Series(np.arange(1, n))
+            operand_scalar = 10
+            pd.testing.assert_series_equal(
+                hpat_func(operand_series, operand_scalar),
+                test_impl(operand_series, operand_scalar),
+                check_names=False)
+
+    def test_series_op8_float_scalar(self):
+        comparison_methods = ('lt', 'gt', 'le', 'ge', 'ne', 'eq')
```

Comment:
Please, remove ne method from the test or implement functional to make 'ne' with scalars

Reply:
@akharche What wrong with `ne` method here? Just for better understanding

Reply:
@shssf Test fails because we can compare only series and series (lhs._data != rhs._data). So, I suggest to implement the same functionality for numbers in 'ne' method (ex. lhs._data != rhs) or remove 'ne' from the test for this pr